gcloud/test__helpers.py

```diff
@@ -181,14 +181,14 @@ def test_no_environment(self):
 
 
 class Test__get_default_service_project_id(unittest2.TestCase):
-    temp_config_path = None
     config_path = '.config/gcloud/configurations/'
     config_file = 'config_default'
 
     def setUp(self):
         import tempfile
         import os
-        self.temp_config_path = tempfile.gettempdir()
+
+        self.temp_config_path = tempfile.mkdtemp()
```

Comment:
It passed on Appveyor.  Putting the `.config` directory in a randomly-named temporary directory is belt-and-suspenders against multiple processes running on the same machine.  I found the problem because the coverage was < 100% at line 206 below (the branch, not the line).